features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/notifications/edit/EditDefaultNotificationSettingPresenter.kt

```diff
@@ -135,7 +135,13 @@ class EditDefaultNotificationSettingPresenter @AssistedInject constructor(
         suspend {
             // On modern clients, we don't have different settings for encrypted and non-encrypted rooms (Legacy clients did).
             notificationSettingsService.setDefaultRoomNotificationMode(isEncrypted = true, mode = mode, isOneToOne = isOneToOne).getOrThrow()
-            notificationSettingsService.setDefaultRoomNotificationMode(isEncrypted = false, mode = mode, isOneToOne = isOneToOne).getOrThrow()
+            val result = notificationSettingsService.setDefaultRoomNotificationMode(isEncrypted = false, mode = mode, isOneToOne = isOneToOne)
+
+            if (result.isFailure) {
+                result.exceptionOrNull()?.printStackTrace()
+            }
```

Comment:
We cannot use `printStackTrace()`, it will not print the string in the log. Also, if `result.isFailure` is `true`, `exceptionOrNull()` will not be null. You can replace by:

```kotlin
.onFailure {
    Timber.e(it, "Failed to set default notification mode for non-encrypted rooms")
}
```

Reply:
Probably a leftover from some debugging, I think it can be removed since the actual error is catched in `RustNotificationSettingsService`